pkg/k8s/constants.go

```diff
@@ -68,6 +68,7 @@ const (
 	NamespaceEnvVar                      = "NAMESPACE"
 	ImmediateReconcileRequestsBufferSize = 10000
 	MaxNumResourceWatcherRoutines        = 10000
+	ManagerNamespaceSuffixLabel          = "cnrm.cloud.google.com/manager-namespace-suffix"
```

Comment:
Suggest using a new field to CCC `spec` instead of labels. Spec is specifically for user configurations, and it is more managable and traceble compared to labels.

Reply:
Are you suggesting `spec` on CCC (ConfigConnectorContext)? There should be a global flag that indicates:
1. If managers need to run in separate namespaces
2. How to name the namespaces )what should be the suffix of the manager's namespace)
This suggest that CC (ConfigConnector) should have the `ManagerNamespaceSuffix`